benchmarks/pom.xml

```diff
@@ -16,11 +16,15 @@
     <version>0.1.0-SNAPSHOT</version>
   </parent>
 
+  <properties>
+    <maven.javadoc.skip>true</maven.javadoc.skip>
+  </properties>
+
```

Comment:
In my opinion we do not want to generate or export the implementation java documentation. We should generate and export the API documentation but the internal documentation is not necessary to be exported (I would say that can actually make our life harder because users may start to depend on the implementation rather than on the API.)